pagopa/pn-frontend
# PN-12148 - Use selected language on every page of pf-login

## Short description
Load lang for every page on pf-login

## List of changes proposed in this pull request
- Moved initial set of language on App.tsx 

## How to test
Start pf-login, try to change language and then navigate to `auth/login/error`. You should see the the error message translated
